server/bastionlab_polars/src/lib.rs

```diff
@@ -276,6 +279,59 @@ Reason: {}",
         dfs.insert(identifier.clone(), df);
         identifier
     }
+
+    fn persist_df(&self, identifier: &str) -> Result<(), Status> {
```

Comment:
You must check that the dataframe is fetchable before saving it
In the future we will have a specific rule for persistence but for now we rely on "fetchability".

Also, I think they are utility methods implemented for the state to get dfs, you should use them.

Reply:
Added fetchable check.

Regarding the methods implemented to get dataframes. I am unable to use them because I do not store dataframes but dataframe artifacts. If I were to save only dataframes then I would not be able to reinsert them with the same parameters on server restart.